components/installer/README.md

```diff
@@ -120,12 +120,20 @@ Helm installation.
 
 ## Install selected components only
 
-By default, this tool installs a predefined component package. If necessary, you can specify which components to install by customizing the component list located in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file. Available components:
-
-- cluster-essentials
-- istio
-- prometheus-operator
-- provision-bundles
-- dex
-- core
-- remote-environments
\ No newline at end of file
+This tool installs components specified in installation CR. Please see CR template in the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file for more details. 
```

Comment:
... ``the `Installation` CR. See the [installer-cr.yaml.tpl](../../installation/resources/installer-cr.yaml.tpl) file for more details. ``

Reply:
Done